packages/iTwinUI-react/stories/core/Tile.stories.tsx

```diff
@@ -178,7 +177,6 @@ export const Actionable: Story<TileProps> = (props) => {
       metadata={metadata}
       badge={badge}
       thumbnail={thumbnail}
-      buttons={buttons}
       leftIcon={leftIcon}
       rightIcon={rightIcon}
       isSelected={selected}
```

Comment:
Need to remove `menuOptions` and other things that Mayank mentioned.

Reply:
Removed